ingest/user_de.py

```diff
@@ -1,15 +1,18 @@
 import pandas as pd
 import numpy as np
 import sys
+import csv
 
 # input file name as CLI
 # example:
-# python ingest/validate_de.py /Users/mvelyuns/scp-ingest-pipeline/ingest/pairwise.csv
+# python ingest/user_de.py /Users/mvelyuns/scp-ingest-pipeline/ingest/pairwise.csv
 
 if len(sys.argv) > 1:
     file_path = sys.argv[1]
 else:
     file_path = "ingest/pairwise.csv"   
+    #file_path = "ingest/one_vs_rest.csv"   
```

Comment:
Before SCP-5086 and SCP-5087 are marked done, let's refactor `user_de.py` to remove this block and be called more like `de.py` and the other modules in Ingest Pipeline.  In other words, we'll need to ensure your `user_de.py` code can be called via a terminal command to `ingest_pipeline.py` that includes something like those CLI arguments we discussed late last week (i.e., something like the [example here](https://github.com/broadinstitute/scp-ingest-pipeline/blob/development/ingest/ingest_pipeline.py#L56), though the particular arguments will differ).

I don't think that is essential for this PR.  But I do think we'll need it before these two tickets are complete, because a main intent of them is to unblock Rails.